projects/lib/src/oauth-service.ts

```diff
@@ -282,6 +282,15 @@ export class OAuthService extends AuthConfig implements OnDestroy {
         return lcUrl.startsWith('https://');
     }
 
+    protected assertUrlNotNullAndCorrectProtocol(url: string | undefined, description: string) {
+      if (url) {
```

Comment:
Shouldn't that be `if (!url)`, given the error message?

Reply:
right